src/Storages/RocksDB/EmbeddedRocksDBBulkSink.cpp

```diff
@@ -169,14 +168,49 @@ std::pair<ColumnString::Ptr, ColumnString::Ptr> EmbeddedRocksDBBulkSink::seriali
         WriteBufferFromVector<ColumnString::Chars> writer_key(serialized_key_data);
         WriteBufferFromVector<ColumnString::Chars> writer_value(serialized_value_data);
         FormatSettings format_settings; /// Format settings is 1.5KB, so it's not wise to create it for each row
+
+        /// TTL handling
+        [[maybe_unused]] auto encode_fixed_32 = [](char * dst, int32_t value)
+        {
+            if constexpr (std::endian::native == std::endian::little)
+            {
+                memcpy(dst, &value, sizeof(value));
+            }
+            else
+            {
+                dst[0] = value & 0xff;
+                dst[1] = (value >> 8) & 0xff;
+                dst[2] = (value >> 16) & 0xff;
+                dst[3] = (value >> 24) & 0xff;
+            }
+        };
```

Comment:
Can we use `writeBinaryEndian` from `WriteHelpers`?

https://github.com/ClickHouse/ClickHouse/blob/ab0968740c6cd2ae4edd3601d7df03ff2970c192/src/IO/WriteHelpers.h#L1370-L1386